### PR TITLE
BOAT-1058 API - Create audit event document copy update

### PR DESCRIPTION
**Pull request by glpins**
## Describe your changes

[BOAT-1058](https://pins-ds.atlassian.net/browse/BOAT-1058)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/api/src/server/endpoints/documents/documents.controller.js [hunk 1]
```diff
@@ -210,20 +213,35 @@ const updateDocuments = async (req, res) => {
 			document.latestVersion = latestDocument?.latestDocumentVersion?.version;
 
 			if (latestDocument && latestDocument.name) {
-				const auditTrail = await createAuditTrail({
-					appealId: appeal.id,
-					azureAdUserId: req.get('azureAdUserId'),
-					details: stringTokenReplacement(AUDIT_TRAIL_DOCUMENT_REDACTED, [
+				if (document.redactionStatus !== latestDocument?.latestDocumentVersion?.redactionStatusId) {
```
Comment (james-hill-kc) [inline appeals/api/src/server/endpoints/documents/documents.controller.js:216]: It looks like this conditional is duplicated

**Review by james-hill-kc — COMMENTED** [review]
Looks good overall, just a couple of minor comments

File: appeals/api/src/server/endpoints/documents/documents.controller.js (hunk 1 above)
Comment (glpins) [reply appeals/api/src/server/endpoints/documents/documents.controller.js:216]: good catch, maybe a bad rebase ??

**Comment by AlexMoulin** [comment]
Everything else looks good otherwise :)

**Review by AlexMoulin — APPROVED** [review]

**Pull request merged**